cpp/src/io/parquet/page_data.cu

```diff
@@ -579,9 +579,20 @@ void write_final_offsets(host_span<size_type const> offsets,
                          host_span<size_type* const> buff_addrs,
                          rmm::cuda_stream_view stream)
 {
-  // Copy offsets to device and create an iterator
+  auto host_pinned_offsets = cudf::detail::make_host_vector<size_type>(offsets.size(), stream);
+  auto host_pinned_buff_addrs = cudf::detail::make_host_vector<size_type*>(buff_addrs.size(), stream);
```

Comment:
As discussed on Slack, we’re using `host_vector` to eliminate pageable copies by setting `LIBCUDF_ALLOCATE_HOST_AS_PINNED_THRESHOLD` to a sufficiently high value.

I was wondering whether we should explicitly use `make_pinned_vector(size, stream)` here instead.

Reply:
Fixed in `cpp/src/io/parquet/reader_impl.cpp`